apps/sms/views/conversation/js/startup.js

```diff
@@ -72,19 +62,17 @@
   exports.Startup = {
     init() {
       MessageManager.init();
-      Navigation.init();
       ConversationView.init();
-
-      ConversationView.once('visually-loaded', () => {
+      if (Navigation.isDefaultPanel()) {
```

Comment:
it means we're in a subpanel (in our case: composer, report, or group)
main panel is a conversation (and needs a threadId)